README.md

```diff
@@ -640,7 +640,7 @@ If you are using an output template inside a Windows batch file then you must es
 
 #### Output template examples
 
-Note that on Windows you may need to use double quotes instead of single.
+Note that on Windows you may need to use double quotes instead of single quotes.
```

Comment:
This kinda change makes 0 sense.

Reply:
Else it is "single" without specify what, so this should either be "double quotes instead of single quotes" or "double instead of single quotes".

Reply:
I dunno if that's a language rule, but one must be stupid to be confused with this line.